code/modules/mob/say.dm

```diff
@@ -76,11 +75,21 @@
 	if(speaking.flags & INNATE)
 		return TRUE
 
-	//Language check.
+	// Language check.
 	for(var/datum/language/L in src.languages)
 		if(speaking.name == L.name)
 			return TRUE
 
+	// Try to translate with augment if it is installed and not broken.
+	if(ishuman(src))
```

Comment:
best practice is not to use istype src checks, instead you should do something like this:

```
/mob/living/carbon/human/say_understands(var/mob/other, var/datum/language/speaking = null)
	. = ..()
	if(!.)
		your translator code here
```